.cirrus.yml

```diff
@@ -0,0 +1,40 @@
+task:
+  trigger_type: manual
+  matrix:
+    - name: E2E/Integration Tests
+      timeout_in: 45
+      container:
+        image: ubuntu:20.04
+        kvm: true # required for E2E/Integrations tests
+        cpu: 4
+        memory: 8G
+      env:
+        DEBIAN_FRONTEND: noninteractive
+        CI: true
+        FORCE_COLOR: 1 # force terminal color
+        CIRRUS_CLONE_DEPTH: 0 # Required to get app version
+        PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD: 1 # disable auto-browser download
+      install_deps_script: |
+        apt-get update
+        apt-get install -y --no-install-recommends ca-certificates gcc g++ curl git golang openssh-client make netcat sudo vim
+        curl -fsSL https://deb.nodesource.com/setup_16.x | sudo -E bash - && apt-get install -y nodejs && node --version && npm -v
```

Comment:
Umm, isn't there a saner way of installing Node? `curl | bash` is a terrible idea.

Reply:
The saner way to install it is through `brew`, but need to be installed brew via `curl` on Linux... so still the same way...
The cirrus ubuntu container just contains essential tools for booting, so we need to install node and other tools somehow.
Do you have any other suggestion to install node in a saner way?